app/blog/20161005-frontend-build-instructions

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
You'll have to add `.static.md` to the end of the file name for it to render on the site